docs/CodeDevelopment/rust_build.md

```diff
@@ -39,13 +39,25 @@ It is recommended to:
 
       \>`rustup component add rust-src --toolchain 1.73.0-x86_64-unknown-linux-gnu`
 
-5. Install `cargo make`
+5. Install `cargo binstall`
 
-   \>`cargo install --force cargo-make`
+   - Windows:

```

Comment:
It would keep things up-to-date with less effort if it links to the [installation section](https://github.com/cargo-bins/cargo-binstall?tab=readme-ov-file#installation) of cargo-binstall which currently provides the same commands.

Reply:
good Idea. Thank you